src/pages/Tokens/views/Balances.tsx

```diff
@@ -68,6 +74,24 @@ export const TokensBalances: () => JSX.Element = () => {
     };
   }, []);
 
+  const fetchPool = async (
```

Comment:
suggestions for a place I can put this common function? Currently used in 2 places

Reply:
I think the `utils` folder would be a good place